ontology/yaml/resources/HVAC/entity_types/PMP.yaml

```diff
@@ -119,6 +119,18 @@ PMP_SS_CWCS_VSFC:
   - run_mode
   - lost_power_alarm
 
+PMP_SS_RMM:
+  description: "RO Pump with pressure switch and failed alarm monitoring"
```

Comment:
can remove RO. is not relevant to the type.

Reply:
Change done